layouts/partials/nav.html

```diff
@@ -3,6 +3,9 @@
 <nav id="nav">
     <ul class="links">
         <li class="active"><a href='{{ "/" | relLangURL }}'>{{ .Site.Title }}</a></li>
+        {{ range .Site.Data.navigation.links }}
+            <li><a href="{{ .url | relURL }}" {{ if in .url "http" }}target="_blank"{{ end }}>{{ .title }}</a></li>
```

Comment:
I'm not 100% sure about having the `target=blank` for external URLs. Some instances might want this to be the same windows (for example this theme could be used as a documentation site which links to another application under a different domain).

Perhaps the target=blank could be a `newwindow` config instead to allow flexibility?